samples/AzureFunctionsUnitTests/SampleUnitTests.cs

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Company.Function; // same namespace as the Azure Functions app
+
+using System.IO;
+using System.Net;
+using System.Security.Claims;
+using System.Text;
+using System.Text.Json;
+using Azure.Core.Serialization;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+
+public class SampleUnitTests
+{
+    [Fact]
+    public async Task OrchestrationReturnsMultipleGreetings()
+    {
+        // create mock orchestration context, and mock ILogger.
+        var contextMock = new Mock<TaskOrchestrationContext>();
```

Comment:
Small style nit: we predominately use inferred `new` in this repo:

```suggestion
        Mock<TaskOrchestrationContext> contextMock = new();
```